### PR TITLE
applications: nrf_desktop: Change connection parameters update timeout to 1s

### PR DESCRIPTION
**Pull request by ppryga**
When peripheral(mouse) is connected to master device, connection parameters are
updated after about 5s, that ends with scattering cursor for the period. To avoid that
timeout for parameters update is decreased to 1s.

Jira:DESK-606

Signed-off-by: Piotr Pryga <piotr.pryga@nordicsemi.no>

### CODE REVIEW

**Comment by ppryga** [comment]
@MarekPieta provide a review please

File: applications/nrf_desktop/configuration/nrf52840_pca20041/app_ZDebugWithShell.conf [hunk 1]
```diff
@@ -204,6 +204,7 @@ CONFIG_BT_GATT_HIDS_ATTR_MAX=19
 CONFIG_BT_GATT_HIDS_DEFAULT_PERM_RW_ENCRYPT=y
 
 CONFIG_BT_CONN_CTX=y
+CONFIG_BT_CONN_PARAM_UPDATE_TIMEOUT=1000CONFIG_BT_CONN_PARAM_UPDATE_TIMEOUT=1000
```
Comment (MarekPieta) [inline applications/nrf_desktop/configuration/nrf52840_pca20041/app_ZDebugWithShell.conf]: Add config once :)

Comment (ppryga) [reply applications/nrf_desktop/configuration/nrf52840_pca20041/app_ZDebugWithShell.conf]: done

**Comment by ppryga** [comment]
@MarekPieta I've changed default timout only for mouses because only there is has visible effect. I sugges to avoid changes that are not required and do not provide measurable value.

**Pull request merged**